File: data/multilang_comparison/program_c/src/func49.c
#include "program_c.h" 
array_t* func49(array_t_param* vars, int loopsFactor) {
   size_t pCounter = vars->size;
   array_t* array15;
   if (pCounter > 0) {
      array15 = vars->data[--pCounter];
      array15->refC++;
      DEBUG_COPY(array15->id);
   } else {
      array15 = (array_t*)malloc(sizeof(array_t));
      array15->size = 873;
      array15->refC = 1;
      array15->id = 15;
      array15->data = (unsigned int*)malloc(array15->size*sizeof(unsigned int));
      memset(array15->data, 0, array15->size*sizeof(unsigned int));
      DEBUG_NEW(array15->id);
   }
   return array15;
}

File: data/multilang_comparison/program_julia/src/func49.jl
include("program_julia_head.jl")
function func49(vars::JArray_param, loopsFactor::Int)::JArray
   pCounter = length(vars.data)
   array15 = nothing
   if pCounter > 1
       array15 = vars.data[pCounter]
       pCounter -= 1
       array15.refC += 1
   else
       array15 = JArray(zeros(Int, 873), 873, 1, 15)
   end
   return array15;
end

